files/es/learn/common_questions/how_does_the_internet_work/index.html

```diff
@@ -80,9 +80,9 @@ <h3 id="Una_red_de_redes">Una red de redes</h3>
 
 <h3 id="Encontrando_ordenadores">Encontrando ordenadores</h3>
 
-<p>Si deseas enviar un mensaje a una computadora, debes especificar a cuál. Es por ello que todo ordenador conectado a una red cuenta con una dirección única que lo identifica, llamada “dirección IP” o Protocolo de Internet(IP de sus siglas en inglés <em>Internet Protocol</em>). Esta dirección está compuesta por una serie de cuatro números separados por puntos, por ejemplo: <code>192.168.2.10</code>.</p>
+<p>Si deseas enviar un mensaje a una computadora, debes especificar a cuál. Es por ello que todo ordenador conectado a una red cuenta con una dirección única que lo identifica, llamada “dirección IP” (siendo "IP" las siglas en inglés de <em>Internet Protocol</em>, o <em>Protocolo de Internet</em>). Esa dirección está compuesta por una serie de cuatro números separados por puntos, por ejemplo: <code>192.168.2.10</code>.</p>
```

Comment:
* "esta" ya que hablamos de algo de aquí en el ejemplo y no de allá en otro lugar

```suggestion
<p>Si deseas enviar un mensaje a una computadora, debes especificar a cuál. Es por ello que todo ordenador conectado a una red cuenta con una dirección única que lo identifica, llamada “dirección IP” (siendo "IP" las siglas en inglés de <em>Internet Protocol</em>, o <em>Protocolo de Internet</em>). Esta dirección está compuesta por una serie de cuatro números separados por puntos, por ejemplo: <code>192.168.2.10</code>.</p>
```

Reply:
M, ok, yo, de algún modo, en mi español rioplatense, tengo un problema con esa redacción, pero es mi problema, no del sitio web... :-)

Reply:
Se intenta utilizar un español que sea entendible para absolutamente todos los países hispanohablantes, es un trabajo complicado, pero lo intentamos, igual si hay algo que no hace sentido en tu español siententé libre de comunicarlo. Gracias